package.json

```diff
@@ -43,7 +43,8 @@
   "activationEvents": [
     "onLanguage:scala",
     "workspaceContains:build.sbt",
-    "workspaceContains:project/build.properties"
+    "workspaceContains:project/build.properties",
+    "workspaceContains:**/scala/**"
```

Comment:
@gabro @olafurpg what about this? It would work for both gradle and maven, which could normally also not have a scala source.

Reply:
I think this would generate many false positives with other builds tools Metals does not support, but it's not the end of the world, since Metals would bail anyway.

Out of curiosity, is there any way we can be more specific to Gradle and Maven?

Reply:
>> I think this would generate many false positives with other builds tools Metals does not support, but it's not the end of the world, since Metals would bail anyway.

But if someone opens a project and it doesn't have any supported build tool then the message about the import should not pop up I think, so shouldn't be an issue.

>> Out of curiosity, is there any way we can be more specific to Gradle and Maven?

We would need to somehow parse the files or run the build tool, which I don't think is possible here.

Reply:
👍 fine by me, then!